crates/rooch-framework/sources/crypto/ecdsa_k1.move

```diff
@@ -2,32 +2,32 @@ module rooch_framework::ecdsa_k1 {
     use std::vector;
 
     /// constant codes
-    const V_ECDSA_K1_TO_BITCOIN_SCHEME_LENGTH: u64 = 1;
-    const V_ECDSA_K1_COMPRESSED_PUBKEY_LENGTH: u64 = 33;
-    const V_ECDSA_K1_SIG_LENGTH: u64 = 64;
+    const VALID_ECDSA_K1_TO_SCHEME_BITCOIN_LENGTH: u64 = 1;
```

Comment:
Do we still need the `VALID_` prefix?

Reply:
> Do we still need the `VALID_` prefix?

Yes until we define the error map you mentioned above and then we can remove `VALID_` prefix.